packages/extensions/core/resources/plugin-trenton.md

```diff
@@ -1,7 +1,7 @@
-# Default Configuration - trenton
```

Comment:
This file needs to be renamed to `plugin-terraform.md`, seems to be the reason for the tests failing